acceptance/openstack/compute/v2/servers_test.go

```diff
@@ -11,170 +10,91 @@ import (
 	"github.com/gophercloud/gophercloud/openstack"
 	"github.com/gophercloud/gophercloud/openstack/compute/v2/servers"
 	"github.com/gophercloud/gophercloud/openstack/networking/v2/networks"
-	"github.com/gophercloud/gophercloud/pagination"
 	th "github.com/gophercloud/gophercloud/testhelper"
 )
 
-func TestListServers(t *testing.T) {
+func TestServersList(t *testing.T) {
 	client, err := newClient()
 	if err != nil {
 		t.Fatalf("Unable to create a compute client: %v", err)
 	}
 
-	t.Logf("ID\tRegion\tName\tStatus\tIPv4\tIPv6")
-
-	pager := servers.List(client, servers.ListOpts{})
-	count, pages := 0, 0
-	pager.EachPage(func(page pagination.Page) (bool, error) {
-		pages++
-		t.Logf("---")
-
-		servers, err := servers.ExtractServers(page)
-		if err != nil {
-			return false, err
-		}
-
-		for _, s := range servers {
-			t.Logf("%s\t%s\t%s\t%s\t%s\t\n", s.ID, s.Name, s.Status, s.AccessIPv4, s.AccessIPv6)
-			count++
-		}
-
-		return true, nil
-	})
-
-	t.Logf("--------\n%d servers listed on %d pages.\n", count, pages)
-}
-
-func networkingClient() (*gophercloud.ServiceClient, error) {
-	opts, err := openstack.AuthOptionsFromEnv()
+	allPages, err := servers.List(client, servers.ListOpts{}).AllPages()
 	if err != nil {
-		return nil, err
+		t.Fatalf("Unable to retrieve servers: %v", err)
 	}
 
-	provider, err := openstack.AuthenticatedClient(opts)
+	allServers, err := servers.ExtractServers(allPages)
 	if err != nil {
-		return nil, err
+		t.Fatalf("Unable to extract servers: %v", err)
 	}
 
-	return openstack.NewNetworkV2(provider, gophercloud.EndpointOpts{
-		Region: os.Getenv("OS_REGION_NAME"),
-	})
+	for _, server := range allServers {
+		printServer(t, &server)
+	}
 }
 
-func createServer(t *testing.T, client *gophercloud.ServiceClient, choices *ComputeChoices) (*servers.Server, error) {
-	if testing.Short() {
-		t.Skip("Skipping test that requires server creation in short mode.")
+func TestServersCreateDestroy(t *testing.T) {
+	client, err := newClient()
+	if err != nil {
+		t.Fatalf("Unable to create a compute client: %v", err)
```

Comment:
Short answer: for errors, I think the output is the same for both, so I'm indifferent.

Longer answer: in general, `%s` is for formatting strings, so using it with an `error` will call the error's `Error() string` method (the only method required to satisfy the `error` interface). `%v` (notably `%+v`) is useful for collection types (`map`, `struct`), where the '+' also prints the field names (if any).